### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of gtk-rs/gir#806.

cc @EPashkin @sdroege

### CODE REVIEW

**Comment by EPashkin** [comment]
@GuillaumeGomez Thanks, 👍

**Comment by GuillaumeGomez** [comment]
Oh damn, I need to update sys part too.

**Comment by EPashkin** [comment]
Seems no `cargo fmt` now on sys

File: Makefile [hunk 1]
```diff
@@ -22,7 +22,8 @@ regen_check: $(GIR) $(GIR_FILES)
 src/auto/mod.rs : Gir.toml $(GIR) $(GIR_FILES)
 	$(GIR) -c Gir.toml
 
-gir-sys : sourceview-sys/src/lib.rs
+gir-sys: sourceview-sys/src/lib.rs
+	cd sourceview-sys && cargo fmt
```
Comment (EPashkin) [inline Makefile]: IMHO better move `cd sourceview-sys && cargo fmt` from `gir : src/auto/mod.rs`
to `sourceview-sys/src/lib.rs`.
And maybe `cargo fmt` from `gir : src/auto/mod.rs` to `src/auto/mod.rs`

Comment (EPashkin) [reply Makefile]: Or just change `gir : src/auto/mod.rs` to `gir : sourceview-sys/src/lib.rs src/auto/mod.rs`, not sure which is better

Comment (EPashkin) [reply Makefile]: Hm, seems I wrong again, your version do my first suggestion,  but `cd sourceview-sys && cargo fmt` just was unneeded in `gir:` part.

Comment (GuillaumeGomez) [reply Makefile]: I moved it into the rule. Tell me if my change sounds good to you.

Comment (EPashkin) [reply Makefile]: Thanks you, sorry for nagging

Comment (GuillaumeGomez) [reply Makefile]: No you were right, It allowed us to clean some Makefile, which is always a good thing. :)

**Pull request merged**